### ISSUE TITLE
mobile/checkout-as-persistent-user e2e test failing in nightly builds

### ISSUE THREAD

**Issue by KateChuen** [SAP/spartacus#5449]
this test is failing
mobile/checkout/checkout-as-persistent-user-mobile.e2e-spec.ts

From the video, it's failing on the shipping address checkout step, where it did not actually click on Continue to go on the "Shipping Method checkout step".

<img width="1165" alt="Screen Shot 2019-11-18 at 11 25 33 AM" src="https://user-images.githubusercontent.com/45409515/69070466-35f3db00-09f6-11ea-8009-77b7bcfad847.png">

**Comment by qjimmy** [comment]
**REMINDER**: Marcin's fix on `feature/GH-4432-counter-poc` consistently fixes the checkout-as-persistent-user test. Still fails on rare occasions but the change in #5457 should prevent this.

**Comment by giancorderoortiz** [comment]
~Related to https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5730~

**Comment by giancorderoortiz** [comment]
blocked by https://cxjira.sap.com/browse/C3PO-2396

**Comment by Zeyber** [comment]
Not blocked. PRs above fix it. Cypress test was not waiting for occ call.

**Comment by Zeyber** [comment]
Note: Partially fixed by #8101 but there is another issue with the test locking here
![image](https://user-images.githubusercontent.com/11735817/91042130-92cd9300-e611-11ea-89ac-e05a5f50bfad.png)

**Comment by Xymmer** [comment]
passing as of oct2
